src/test/java/TicTacToe/TicTacToeTest.java

```diff
@@ -2,49 +2,52 @@
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.mockito.InOrder;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
-import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
 
 import static org.mockito.Mockito.*;
 
+@ExtendWith(MockitoExtension.class)
 public class TicTacToeTest {
 
-    @Mock
-    private WinLogic winLogic;
     @Mock
     private Board board;
     @Mock
     private UserInterface userInterface;
     @Mock
-    private MovementLogic movementLogic;
+    private PlayOption playOption;
+    @Mock
+    private WinLogic winLogic;
+    @Mock
+    private HumanPlayerFactory humanPlayerFactory;
+    @Mock
+    private ComputerPlayerFactory computerPlayerFactory;
+    @Mock
+    private PlayLogic playLogic;
     @Mock
-    private Position position;
+    private ScannerWrapper scannerWrapper;
     @InjectMocks
     private TicTacToe ticTacToe;
 
     @BeforeEach
```

Comment:
Jak masz @InjectMocks to nie musisz tworzyć sma obiektu w setupie

Reply:
fixed